eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -114,31 +115,39 @@ jobs:
       maxParallel: $[ variables['MaxParallelTestJobs'] ]
       matrix:
         Linux:
+          Pool: # Intentionally blank.
           OSVmImage: "ubuntu-18.04"
           TestTargetFramework: netcoreapp2.1
         Windows_NetCoreApp:
-          OSVmImage: "windows-2019"
+          Pool: azsdk-pool-mms2019-ds2v2-general # Comment out to swap back to public hosted pool.
```

Comment:
I assume this is approach is still for testing correct? Do we have any more concrete plans for what this will look like when we make the switch? It would be great if we could come up with a way to avoid this bifurcation if possible.

Reply:
I'm not sure this is that messy compared with other options which might sprinkle conditional logic all over the place. I think this is probably easier to reason over just by glancing at it.

Reply:
Could we create a pool for every image we use and then just use the pool name everywhere and control using 1ES or Hosted DevOps machines in the pool configuration?

Reply:
As discussed, this isn't something that Azure DevOps supports.